packages/planner/src/app/components_ngrx/planner-board/planner-board.component.html

```diff
@@ -1,71 +1,83 @@
-<div class="container-fluid f8-wi-container" #containerHeight>
-  <div class="lock-overlay-list" *ngIf="(boardUiQuery.boardLocked | async)">
-    <div class="spinner-container">
-      <h2 class="spinner"></h2>
-      <h2>Loading ...</h2>
-      <h4>Please wait, we are loading your data.</h4>
-    </div>
-  </div>
-
-  <main class="row">
-    <alm-planner-layout
-      #plannerLayout
-      (sidePanelStateChange)="togglePanelState($event)"
-      [sidePanelContent]="sidePanel"
-      [sectionContent]="sectionPanel"
-    >
-    </alm-planner-layout>
-    <work-item-preview-panel #quickPreview [context]="'board'"> </work-item-preview-panel>
-  </main>
-</div>
+<f8-feature-toggle
+  featureName="Planner"
```

Comment:
@divyanshiGupta Should board view be on the same feature level as planner? Planner is on beta, i think board should be on same level as Query tab which is internal. @nimishamukherjee What do you think?

Reply:
Yes, I think it should be on the same level as query tab. I will change that.